src/main/java/uk/gov/ons/ssdc/supporttool/utility/CaseSearchResultsMapper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.ons.ssdc.supporttool.utility;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Map;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ssdc.supporttool.model.dto.CaseSearchResult;
+
+@Component
+public class CaseSearchResultsMapper implements RowMapper<CaseSearchResult> {
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+
+  @SneakyThrows
```

Comment:
I know the support tool is a bit more fast and loose, but this has a bad code smell about it